### DISCUSSION TITLE
SiamDW ?

### DISCUSSION THREAD

**Discussion by MohamedAliRashad** [STVIR/pysot#254] [General] [answered]
There is a new Siamese Network for tracking released and accepted in CVPR 2019
https://github.com/researchmm/SiamDW

Is there plans to include it in pysot ?

**Comment by DeepTiming** [comment]
different author

**Comment by MohamedAliRashad** [comment]
@DeepTiming 
I contacted the people working on the repo and they have shown an interest in contributing to pysot
https://github.com/researchmm/SiamDW/issues/63

**Comment by ZhiyuanChen** [comment, marked as answer]
> @DeepTiming
> I contacted the people working on the repo and they have shown an interest in contributing to pysot
> [researchmm/SiamDW#63](https://github.com/researchmm/SiamDW/issues/63)

Actually I don't think what he means is to merge SiamDW to pysot. We are two different company, it's very unlikely that he contributes to our project. However, such contributions will be welcomed and appreciated.
